core/modules/css/EmptyChunks.cc

```diff
@@ -35,71 +35,50 @@
 #include "lsst/log/Log.h"
 
 // Qserv headers
+#include "css/DbInterfaceMySql.h"
+#include "css/CssError.h"
```

Comment:
Headers would need to be swapped to comply with the adopted header sorting rules.